docs/source/Manuals/SManual/SManual.rst

```diff
@@ -11,7 +11,7 @@ Introduction to SimPy
 
 :Authors: G A Vignaux and Klaus Muller
 :Date:  |today|
-:SimPy release: |release|
+:Release: |release|
```

Comment:
Some places say release some say SimPy Classic Release

Reply:
Where I said that the manual title did not have Simpy Classic in it

Reply:
fixed any that needed it